miniconda/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
```

Comment:
What version of openssl is being used? It could also be an issue with `wget`. Does `curl` work?